tests/shared_tests.py

```diff
@@ -0,0 +1,24 @@
+def run_save_load_train(happy, output_path, args, data_path, run_type):
```

Comment:
It seems odd that `run_save_load_train()` doesn't necessarily train anything.

Two thirds of the code paths (`run_path=='eval'` and `run_path=='test'`) avoid training entirely

Reply:
Fixed in 909019e